src/mocking.spec.ts

```diff
@@ -32,7 +32,7 @@ describe('mockInstanceOf', () => {
       build: () => OK
     });
     expect(mockCreep.body[1].type).toEqual(WORK);
-    expect(mockCreep.room.controller?.owner.username).toEqual('some-user');
+    expect(mockCreep.room.controller?.owner!.username).toEqual('some-user');
```

Comment:
This change was to satisfy a type-check error when building locally.

Reply:
Continuing the optional chaining for the potentially undefined property would be more accurate than asserting that it will not be undefined. `mockCreep.room.controller?.owner?.username`